src/jobs/push-to-app-catalog.yaml

```diff
@@ -20,6 +20,10 @@ parameters:
       When this is `false`, commits to `master` will be pushed to `app_catalog` instead of `app_catalog_test`.
       Set this to `false` for deployments that follow a a master branch for production releases rather than
       using tags (the default).
+  policy_path:
+    description: Path to the Rego policy files directory.
+    type: string
+    default: policy
```

Comment:
Can't we use the same policy across all the repos?

Reply:
It will use the same policy across all the repos, unless that repo specify a different `policy_path` parameter containing custom policies.

Reply:
I'm asking if we need it at all. I'd rather add it later on when we have a legit reason for it.

Reply:
I guess it doesn't harm. What do you think @TheoBrigitte ?

Reply:
I'm fine with that

Reply:
I have opinion on that. And I'm pushing providing as lean interface as possible. Then we have more freedom to change and it's easier to keep things aligned.